aws/resource_aws_vpc.go

```diff
@@ -235,6 +241,30 @@ func resourceAwsVpcRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("enable_classiclink", classiclink_enabled)
 	}
 
+	DescribeClassiclinkDnsOpts := &ec2.DescribeVpcClassicLinkDnsSupportInput{
+		VpcIds: []*string{&vpcid},
+	}
+
+	respClassiclinkDnsSupport, err := conn.DescribeVpcClassicLinkDnsSupport(DescribeClassiclinkDnsOpts)
+	if err != nil {
+		if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == "UnsupportedOperation" {
+			log.Printf("[WARN] VPC Classic Link DNS Support is not supported in this region")
+		} else {
+			return err
+		}
```

Comment:
This follows *exactly* the same pattern as `enable_classiclink` - that seems to work well so therefore, we don't set it to default - thoughts?